archstrike/iaito-git/PKGBUILD

```diff
@@ -0,0 +1,36 @@
+# Maintainer: ArchStrike <team@archstrike.org>
+
+buildarch=220
+
+pkgname=iaito-git
+_pkgname=${pkgname::-4}
+pkgver=r254.b7e9afc
+pkgrel=1
+pkgdesc="A Qt and C++ GUI for radare2 reverse engineering framework"
+url="http://www.iaito.re"
+arch=('i686' 'x86_64' 'armv6h' 'armv7h' 'aarch64')
+groups=('archstrike')
+license=('GPL3')
+depends=('qt5-base' 'qt5-tools' 'qt5-webengine' 'radare2')
+makedepends=('git')
+provides=("${_pkgname}")
```

Comment:
provides=("bokken")
replaces=("bokken")
conflicts=("bokken")

to force replacement during upgrade.

no need for provides ${_pkgname}